builds/heroku-16.Dockerfile

```diff
@@ -7,7 +7,7 @@ ENV WORKSPACE_DIR="/app/builds" \
     DEBIAN_FRONTEND=noninteractive \
     STACK="heroku-16"
 
-RUN apt-get update && apt-get install -y python-pip && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y python-pip libsqlite3-dev && rm -rf /var/lib/apt/lists/*
```

Comment:
This should be done inside the python formula, not globally (`apt-get update && apt-get install -y libsqlite3-dev`). See https://github.com/heroku/heroku-buildpack-php/blob/343fa959d4c195beb7febe1996340624baf713b9/support/build/php#L59-L71 for how to do this (the stack specific additions there are of course not relevant here)

Reply:
(also, this is missing from `heroku-18.Dockerfile` right now)